topics/responses.md

```diff
@@ -1,230 +1,147 @@
 [//]: # (title: Responses)
```

Comment:
It feels like "Set response payload" and "Set response parameters", and "Redirect" are just randomly placed without any introduction as to why we're covering this. It would be good to have a brief section discussing types of responses or what can be done and then link to these.

Reply:
![image](https://user-images.githubusercontent.com/38073144/113588529-f18fcc00-9638-11eb-9172-467167cc7099.png)

Added links to sections and info about redirects to a topic intro. I hope, it matches better with a topic TOC now.